### ISSUE TITLE
[Feature Request] Adding wordcount function

### ISSUE THREAD

**Issue by SConv11** [GorvGoyl/Notion-Boost-browser-extension#227]
Adding a wordcount function will be very helpful since the built-in wordcount tool offers little information and is somehow hard to access. Also, the built-in wordcounter feature is extremely unfriendly for foreign language users. E.g. it displays a wrong number if the whole page is written in Chinese. 

That from notion enhancer can serve as reference.

**Comment by 93Alliance** [comment]
Yes, the notion built-in one doesn't display directly and has problems with the stats, I'm currently using the notion mate extension which does this, hopefully the author can implement it.

![image](https://github.com/user-attachments/assets/b9173330-9d15-48d1-a4a4-eafd947d2e63)
